spec/tapioca/cli_spec.rb

```diff
@@ -569,6 +572,95 @@ def body=(body); end
       refute_path_exists("#{outdir}/namespace/comment.rbi")
       refute_path_exists("#{outdir}/user.rbi")
     end
+
+    describe("verify") do
+      describe("with no changes") do
+        before do
+          execute("dsl")
+        end
+
+        it 'does nothing and returns exit_status 0' do
+          output = execute("dsl", "--verify")
+
+          assert_includes(output, <<~OUTPUT)
+            Nothing to do, all RBIs are up-to-date.
+          OUTPUT
+          assert_includes($?.to_s, "exit 0") # rubocop:disable Style/SpecialGlobalVars
```

Comment:
I guess making `execute` return the result code would be a lot of changes?

Reply:
Yeah, would require changing every single test. Previously, I was setting a global variable, but was causing the tests to be flakey. This seems to have resolved the flakiness and didn't require modifying each test.

Reply:
Can we add a method `last_execute_status` to `CliSpec` and store the result during `execute` instead?

Reply:
That's what I was doing before. Due to the async nature of `IO.popen(..)` it wasn't getting set correctly each time and was causing flakiness.

Reply:
I can make the bigger change for all the tests later.